addusers.py

```diff
@@ -356,6 +361,7 @@ def parse_rows(rows, select_user=None):
     admin_project = config.get('auth', 'admin_project')
     auth_url = config.get('auth', 'auth_url')
     nova_version = config.get('nova', 'version')
+    user_column = config.get('header_column', 'user_column')
```

Comment:
Please also update `example_settings.ini` with the example section and these settings.